src/Routes/TemporaryBetaPage/TemporaryBetaPage.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Bullseye, Button, Title } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+const TemporaryBetaPage = () => {
+    const onClick = (event) => {
+        const isMetaKey = event.ctrlKey || event.metaKey || event.which === 2;
+        const url = `${document.baseURI}beta/insights/malware`;
+        isMetaKey ? window.open(url) : (window.location.href = url);
+    };
+
+    return (
+        <Bullseye>
+            <div
+                className="chr-c-navigation__beta-info-modal"
+                style={{ maxWidth: 800 }}
+            >
+                <InfoCircleIcon size="xl" className="info-icon" />
+                <Title headingLevel="h4" size="xl">
+                    {`Malware is only available in our Beta Environment`}
```

Comment:
I understand this is a fallback for the modal, but this fallback solution doesnt seem to have a mock. In the mock provided, the title is dynamic and includes whatever section of malware the user is in, however here it is hard coded. When trying to access in stable env, malware just disappears, and does not show the beta flag (im not sure if this is just something to do with chrome) . This allows you to not make it dynamic, but is it find to handle it like this? It has a similar desired outcome but I just wanna be sure whoever has to agree with this agrees.

Reply:
> I understand this is a fallback for the modal, but this fallback solution doesnt seem to have a mock. 

This fallback does not have a mock because we expected modal would work every time. This issue could only be fixed on Cloud services config + Chrome level. But I think this solution works for the time being, considering Malware will be only in this half release state for a few weeks.

> the title is dynamic and includes whatever section of malware the user is in, however here it is hard coded.

Yes, because you can open modal by clicking on the subpages in the left nav menu (so the title is dynamic). But you can only bypass the modal if you open the app from the "All apps and services" menu where there are no subpages. So you click a link with text "Malware".

> When trying to access in stable env, malware just disappears, and does not show the beta flag (im not sure if this is just something to do with chrome).

It does not show a beta flag because the Jenkins build to cloud services config that added it failed and I'm waiting for platform folks to restart it. So after the build goes through it will add the beta badge.

This really is a mess :smile:

> I just wanna be sure whoever has to agree with this agrees.

This is my own idea to fix the problem with Chrome, which would take IMO long time to fix, and to a customer this probably looks like it's supposed to be this way. If we don't merge this PR there will be no fallback, which means "Something went wrong" page will be shown.

Reply:
ahh ok, I understand your reasoning behind it all. Thanks for going in-depth